src/Akka.Streams.Kafka.Tests/BugFix240SupervisionStrategy.cs

```diff
@@ -31,8 +32,8 @@ public BugFix240SupervisionStrategy(ITestOutputHelper output, KafkaFixture fixtu
         [Fact]
         public async Task SupervisionStrategy_Decider_on_Producer_Upstream_should_work()
         {
-            var topic = CreateTopic(1);
-            var group = CreateGroup(1);
+            var topic = CreateTopicName(1);
```

Comment:
All of these changes in the test classes are cosmetic, making sure that the names are more in step with its intent, creating a topic name and group id, instead vague name and group which can also means actual topic and group